src/throttle/amoc_throttle.erl

```diff
@@ -145,13 +159,6 @@ send(Name, Msg) ->
 send(Name, Pid, Msg) ->
     amoc_throttle_runner:throttle(Name, {Pid, Msg}).
 
-%% @doc Sends and receives the given message `Msg'.
-%%
-%% Deprecated in favour of `wait/1'
--spec send_and_wait(name(), any()) -> ok | {error, any()}.
```

Comment:
I think that this interface is actively used by amoc_arsenal_xmpp. if it's so,  please add tickets for amoc_arsena/amoc_arsenal_xmpp

Reply:
It is already updated 👌🏽